cli/cmd/generate.go

```diff
@@ -64,6 +65,7 @@ subcommands.`,
 	cmd.Flags().StringP("settings", "s", settingsFilename, "path to settings (.json) file")
 	cmd.Flags().StringP("manifest", "m", manifestFilename, "path to manifest (.json) file")
 	cmd.Flags().StringArrayP("workload-owner-key", "w", []string{workloadOwnerPEM}, "path to workload owner key (.pem) file")
+	cmd.Flags().StringArray("seedshare-owner-key", []string{seedshareOwnerPEM}, "path to seedshare owner key (.pem) file")
```

Comment:
```suggestion
	cmd.Flags().StringArrayP("workload-owner-key", "w", []string{workloadOwnerPEM}, "paths to workload owner key (.pem) files")
	cmd.Flags().StringArray("seedshare-owner-key", []string{seedshareOwnerPEM}, "paths to seedshare owner key (.pem) files")
```

Reply:
I would rather add `(can be passed more than once)` or something like that.